package.json

```diff
@@ -28,5 +28,10 @@
     "react-copy-to-clipboard": "^5.1.0",
     "react-dom": "18.2.0",
     "styled-components": "^5.3.6"
+  },
+  "devDependencies": {
+    "@types/react": "^18.0.28",
+    "@types/react-dom": "^18.0.11",
+    "typescript": "^4.9.5"
```

Comment:
once you added `typescript`, add `"tsc": "tsc"` in scripts too. 

so you can run `npm run tsc` to compile/check typescript. 

and also you can run `npm run tsc -- -w` for watch more.